showcase/radar-chart/animated-radar-chart.js

```diff
@@ -72,6 +72,9 @@ export default class AnimatedRadar extends Component {
               text: {
                 opacity: 0
               }
+            },
+            labels: {
```

Comment:
that shouldn't be the desired behavior? the desired effect should that the labels style should read: {...DEFAULT_STYLE, ....USER_PROVIDED_STYLE};

regardless, this PR looks fine ill merge it when the tests pass